content/docs/04.5-devx/05-app-profile/03-app-profile-macros.md

```diff
@@ -13,70 +13,67 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
 import Tooltip from "shared/components/ui/Tooltip";
 
 
-# App Profile Macros
+# Output Variables
 
-Pack macros are the variables defined in the App Profile and are only resolved at the cluster deployment time. The app profile variables can be:
+Palette Dev Engine output variables are defined in the App Profile and are only resolved at cluster deployment time. The output variables have the following properties:
 
-* directly given by the developers while modelling the application profiles.
+* May be referenced by specifying them during app profile creation.
 
-* resolved during the app instatiation.
+* Output variables are inherited from the lower tiers of the app profile.
 
-* inherited from the lower tiers of the App Profile.
+* Each service type exposes a set of unique output variables. 
 
-The variables are of two types:
 
-* [The Input Variables](/devx/app-profile#inputparameters)
 
-* [The Output Variables](/devx/app-profile#outputparameters)
-<br />
-
-## Input Variables
-
-User-provided values that will be consumed by the current tier. It can even use output variables from below tier. This parameter resolves to a value of the variable defined in App Profile tier parameters.yaml file.
-
-```
-{{.spectro.system.[VARIABLE_NAME]}}
-```
-|**Supported Variables**|  Description|
-|-----------------------|-------------|
-|appprofile.name|Name of the app profile|
-|appprofile.uid|Unique id of the app profile|
-|appdeployment.name|Name of the deployment|
-|appdeployment.uid|Unique id of the deployment|
-|appdeployment.tiername| Resolves to a string value in the format `<deployment name>-<tier name>`|
-
-<br />
-
-The parameter which will be generated once the tier gets deployed. It can be consumed by the above tiers. It refers to the tier output variables of the top tiers. Refer the format and example below:
-
-<br />
-
-```
-{{.spectro.system.apptier.<tierInput_Variable_Name>}}
-```
-
-**Example**
-{{.spectro.system.apptier.NAMESPACE}}: 
+The variables are generated once the server layer or tier gets deployed. Output variables can be consumed by the higher tier layers in the app profile. 
 
-
-## Output Variables
-
-The variables are generated once the tier gets deployed. Output variables can be consumed by the higher tiers in the App Profile.
+Check out the [Services Connectivity](/devx/app-profile/services/connectivity) page to learn how to use output variables for establishing network connectivity between services.
 
 
 <br /> 
 
 
 ```
-{{.spectro.app.$appdeploymentName.<tiername>.<tierOutput_Variable_Name>}} 
+{{.spectro.app.$appdeploymentName.<tiername>.<tierOutput_Variable_Name>}}
 ```
 
-**Example**
-{{.spectro.app.$appDepName-mongodb.PASSWORD}}: 
-
-<br />
-
-### Important Links to Refer Palette Macros
+<!-- # System Output Variables
```

Comment:
This is in preparation for the [spreadsheet](https://docs.google.com/spreadsheets/d/1sFQ1WYqmNIpLW607KXCibUxBLV87BwQlb-26VvchUCY/edit#gid=0) the PDE team is filling out for us: